service/worker_slice_config_service.go

```diff
@@ -211,6 +212,22 @@ outer:
 		logger.With(zap.Error(err)).Errorf("Failed to deep copy external gateway configuration")
 	}
 
+	// Reconcile Slice gateway service type
+	sliceGatewayProvider := workerv1alpha1.WorkerSliceGatewayProvider{
```

Comment:
I was expecting this var: sliceGatewayProvider to be set in the workerSliceConfig obj, but dont see it here. 
Shouldnt this be set to workerSliceConfig.Spec.SliceGatewayProvider?

Reply:
yes! Rectified.